App crashes if no answer provided for Assist

<!-- READ THIS FIRST:
- Make sure you run the latest version of the Android app
- Make sure you run the latest version of Home Assistant
- Make sure to check the Companion docs for troubleshooting and configuration: https://companion.home-assistant.io/
- Make sure the bug you found is not already reported, we love to put work in bugfixes instead of closing duplicate bug reports
  DO NOT DELETE ANY TEXT from this template! All requested information is important.
-->

<!-- If you are reporting an issue for Wear OS please list both devices for the below 3 fields -->
**Home Assistant Android app version(s):**

**Android version(s):** 2025.1.2-full

**Device model(s):** Google Pixel 8

**Home Assistant version:** 2025.3.4

**Last working Home Assistant release (if known):**

**Description of problem, include YAML if issue is related to notifications:**

Hi there,
I have the following setup:

/homeassistant/configuration.yaml:

```
# Loads default set of integrations. Do not remove.
default_config:

# Load frontend themes from the themes folder
frontend:
  themes: !include_dir_merge_named themes

# Text to speech
tts:
  - platform: google_translate

automation: !include automations.yaml
script: !include scripts.yaml
scene: !include scenes.yaml

# Connected clients
sensor:
  - platform: websocket_api

intent_script: !include intent_script.yaml

python_script:


```
/homeassistant/custom_sentences/de/intents.yaml:
```
language: "de"
intents:
  VacuumLocate:
    data:
      - sentences:
          - "Wo ist [der] <name>"
        requires_context:
          domain: vacuum
```

/homeassistant/intent_script.yaml
```
VacuumLocate:
  action:
    action: vacuum.locate
    target:
      entity_id: vacuum.roborock_s7
```

Issue:
When typing/speaking the sentence "Wo ist der Staubsauger" (Where is the vacuum cleaner), the intent-script gets executed and after that the app crashes.
I notices that this is because the "speech" in the intend-script is missing - however the app should probably not crash.
<!--
- For Wear OS devices we will need the LogCat logs from the device.
- For Android Auto the logs can be retrieved from the connected device.
- Logs from the device can be taken from Settings > Companion App Troubleshooting > Show and Share Logs
-->
**Companion App Logs:**

Cash-Log
```
Timestamp: 2025-03-30 00:27:26.954
Thread: DefaultDispatcher-worker-9
Exception: java.lang.IllegalArgumentException: Instantiation of [simple type, class io.homeassistant.companion.android.common.data.websocket.impl.entities.ConversationSpeechPlainResponse] value failed for JSON property plain due to missing (therefore NULL) value for creator parameter plain which is a non-nullable type
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: io.homeassistant.companion.android.common.data.websocket.impl.entities.AssistPipelineIntentEnd["intent_output"]->io.homeassistant.companion.android.common.data.websocket.impl.entities.ConversationResponse["response"]->io.homeassistant.companion.android.common.data.websocket.impl.entities.ConversationSpeechResponse["speech"]->io.homeassistant.companion.android.common.data.websocket.impl.entities.ConversationSpeechPlainResponse["plain"])
	at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:4393)
	at com.fasterxml.jackson.databind.ObjectMapper.convertValue(ObjectMapper.java:4324)
	at io.homeassistant.companion.android.common.data.websocket.impl.WebSocketRepositoryImpl.handleEvent(WebSocketRepositoryImpl.kt:752)
	at io.homeassistant.companion.android.common.data.websocket.impl.WebSocketRepositoryImpl.access$handleEvent(WebSocketRepositoryImpl.kt:88)
	at io.homeassistant.companion.android.common.data.websocket.impl.WebSocketRepositoryImpl$onMessage$1$success$1.invokeSuspend(WebSocketRepositoryImpl.kt:884)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:113)
	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:89)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:589)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:823)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:720)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:707)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [LazyStandaloneCoroutine{Cancelling}@e086eb4, Dispatchers.IO]
Caused by: com.fasterxml.jackson.module.kotlin.MissingKotlinParameterException: Instantiation of [simple type, class io.homeassistant.companion.android.common.data.websocket.impl.entities.ConversationSpeechPlainResponse] value failed for JSON property plain due to missing (therefore NULL) value for creator parameter plain which is a non-nullable type
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: io.homeassistant.companion.android.common.data.websocket.impl.entities.AssistPipelineIntentEnd["intent_output"]->io.homeassistant.companion.android.common.data.websocket.impl.entities.ConversationResponse["response"]->io.homeassistant.companion.android.common.data.websocket.impl.entities.ConversationSpeechResponse["speech"]->io.homeassistant.companion.android.common.data.websocket.impl.entities.ConversationSpeechPlainResponse["plain"])
	at com.fasterxml.jackson.module.kotlin.KotlinValueInstantiator.createFromObjectWith(KotlinValueInstantiator.kt:84)
	at com.fasterxml.jackson.databind.deser.impl.PropertyBasedCreator.build(PropertyBasedCreator.java:202)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:519)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1405)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:352)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:185)
	at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:542)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeWithErrorWrapping(BeanDeserializer.java:564)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:439)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1405)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:352)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:185)
	at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:542)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeWithErrorWrapping(BeanDeserializer.java:564)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:439)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1405)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:352)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:185)
	at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:542)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeWithErrorWrapping(BeanDeserializer.java:564)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:439)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1405)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:352)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:185)
	at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:4388)
	... 12 more
```

Current Log:
```
--------- beginning of system
03-30 00:27:28.296 32035 32035 V ResourcesManager: The following library key has been added: ResourcesKey{ mHash=45101bf1 mResDir=null mSplitDirs=[] mOverlayDirs=[/product/overlay/NavigationBarModeGestural/NavigationBarModeGesturalOverlay.apk,/data/resource-cache/com.android.systemui-neutral-mMDp.frro,/data/resource-cache/com.android.systemui-accent-T96X.frro,/data/resource-cache/com.android.systemui-dynamic-j4cf.frro] mLibDirs=[/data/app/~~RklyoWcbQjUGiKDQ1Q1_uA==/com.google.android.webview-6hq7-Tq1MqRYaxZQjoOVBg==/base.apk,/data/app/~~L8hzbes-1c_-Hr1FFTArZw==/com.google.android.trichromelibrary_704903843-P_U8Qj0UBv8ows5nS5VAwA==/base.apk,/system_ext/framework/androidx.window.extensions.jar] mDisplayId=0 mOverrideConfig=v35 mCompatInfo={420dpi always-compat} mLoaders=[]}
03-30 00:29:34.469 32035 32035 D InsetsController: Setting requestedVisibleTypes to -1 (was -9)
--------- beginning of main
03-30 00:41:29.820 32035 32035 D SensorReceiver: Received intent: android.os.action.DEVICE_IDLE_MODE_CHANGED
03-30 00:41:29.821 32035 32035 D SensorReceiver: Sensor(s) [is_idle] corresponding to received event android.os.action.DEVICE_IDLE_MODE_CHANGED are disabled, skipping sensors update
03-30 00:45:43.832 32035 32054 D WM-Processor: Processor cancelling cb306a1c-29cb-4e28-aea4-407c7fa1eb38
03-30 00:45:43.832 32035 32054 D WM-Processor: WorkerWrapper could not be found for cb306a1c-29cb-4e28-aea4-407c7fa1eb38
03-30 00:45:43.834 32035 32054 D WM-GreedyScheduler: Cancelling work ID cb306a1c-29cb-4e28-aea4-407c7fa1eb38
03-30 00:45:43.839 32035 32035 D SensorReceiver: Received intent: android.intent.action.SCREEN_ON
03-30 00:45:43.839 32035 32035 D SensorReceiver: Sensor(s) [is_interactive] corresponding to received event android.intent.action.SCREEN_ON are disabled, skipping sensors update
03-30 00:45:43.845 32035 32035 D ButtonWidget: Broadcast received: 
03-30 00:45:43.845 32035 32035 D ButtonWidget: Broadcast action: android.intent.action.SCREEN_ON
03-30 00:45:43.845 32035 32035 D ButtonWidget: AppWidgetId: -1
03-30 00:45:43.848 32035 32054 D WM-SystemJobScheduler: Scheduling work ID 73453a33-9f0e-460e-8be5-ba3029fd4753Job ID 153960
03-30 00:45:43.851 32035 32035 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:43.853 32035 32054 D WM-GreedyScheduler: Starting work for 73453a33-9f0e-460e-8be5-ba3029fd4753
03-30 00:45:43.853 32035 32035 D MediaPlayCtrlsWidget: Broadcast received: 
03-30 00:45:43.853 32035 32035 D MediaPlayCtrlsWidget: Broadcast action: android.intent.action.SCREEN_ON
03-30 00:45:43.853 32035 32035 D MediaPlayCtrlsWidget: AppWidgetId: -1
03-30 00:45:43.854 32035 32054 D WM-Processor: Processor: processing WorkGenerationalId(workSpecId=73453a33-9f0e-460e-8be5-ba3029fd4753, generation=0)
03-30 00:45:43.860 32035 32035 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:43.860 32035 32035 D WM-SystemJobService: onStartJob for WorkGenerationalId(workSpecId=73453a33-9f0e-460e-8be5-ba3029fd4753, generation=0)
03-30 00:45:43.861 32035 32035 D WM-WorkerWrapper: Starting work for io.homeassistant.companion.android.websocket.WebsocketManager
03-30 00:45:43.861 32035 32054 D WM-Processor: Work WorkGenerationalId(workSpecId=73453a33-9f0e-460e-8be5-ba3029fd4753, generation=0) is already enqueued for processing
03-30 00:45:43.864 32035 32417 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:43.864 32035 32058 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=73453a33-9f0e-460e-8be5-ba3029fd4753, tags={ io.homeassistant.companion.android.websocket.WebsocketManager } ]
03-30 00:45:43.867 32035 32035 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:43.868 32035 32035 D WM-Processor: Processor 73453a33-9f0e-460e-8be5-ba3029fd4753 executed; reschedule = false
03-30 00:45:43.868 32035 32035 D WM-SystemJobService: 73453a33-9f0e-460e-8be5-ba3029fd4753 executed on JobScheduler
03-30 00:45:43.872 32035 32054 D WM-GreedyScheduler: Cancelling work ID 73453a33-9f0e-460e-8be5-ba3029fd4753
03-30 00:45:43.874 32035 32054 D WM-SystemJobScheduler: Scheduling work ID 73453a33-9f0e-460e-8be5-ba3029fd4753Job ID 153960
03-30 00:45:45.884 32035 32035 D SensorReceiver: Received intent: android.os.action.DEVICE_IDLE_MODE_CHANGED
03-30 00:45:45.885 32035 32035 D SensorReceiver: Sensor(s) [is_idle] corresponding to received event android.os.action.DEVICE_IDLE_MODE_CHANGED are disabled, skipping sensors update
03-30 00:45:45.889 32035 32035 D SensorReceiver: Received intent: android.os.action.POWER_SAVE_MODE_CHANGED
03-30 00:45:45.889 32035 32054 D WM-Processor: Processor cancelling 73453a33-9f0e-460e-8be5-ba3029fd4753
03-30 00:45:45.889 32035 32054 D WM-Processor: WorkerWrapper could not be found for 73453a33-9f0e-460e-8be5-ba3029fd4753
03-30 00:45:45.889 32035 32035 D SensorReceiver: Sensor(s) [power_save] corresponding to received event android.os.action.POWER_SAVE_MODE_CHANGED are disabled, skipping sensors update
03-30 00:45:45.891 32035 32054 D WM-GreedyScheduler: Cancelling work ID 73453a33-9f0e-460e-8be5-ba3029fd4753
03-30 00:45:45.924 32035 32054 D WM-SystemJobScheduler: Scheduling work ID fcbd34f2-abe8-40ae-ae07-efc23a911edeJob ID 153961
03-30 00:45:45.933 32035 32054 D WM-GreedyScheduler: Starting work for fcbd34f2-abe8-40ae-ae07-efc23a911ede
03-30 00:45:45.933 32035 32054 D WM-Processor: Processor: processing WorkGenerationalId(workSpecId=fcbd34f2-abe8-40ae-ae07-efc23a911ede, generation=0)
03-30 00:45:45.935 32035 32035 D WM-SystemJobService: onStartJob for WorkGenerationalId(workSpecId=fcbd34f2-abe8-40ae-ae07-efc23a911ede, generation=0)
03-30 00:45:45.935 32035 32035 D WM-WorkerWrapper: Starting work for io.homeassistant.companion.android.websocket.WebsocketManager
03-30 00:45:45.937 32035 32057 D WM-Processor: Work WorkGenerationalId(workSpecId=fcbd34f2-abe8-40ae-ae07-efc23a911ede, generation=0) is already enqueued for processing
03-30 00:45:45.939 32035 32069 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:45.940 32035 32057 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=fcbd34f2-abe8-40ae-ae07-efc23a911ede, tags={ io.homeassistant.companion.android.websocket.WebsocketManager } ]
03-30 00:45:45.943 32035 32035 D WM-Processor: Processor fcbd34f2-abe8-40ae-ae07-efc23a911ede executed; reschedule = false
03-30 00:45:45.943 32035 32035 D WM-SystemJobService: fcbd34f2-abe8-40ae-ae07-efc23a911ede executed on JobScheduler
03-30 00:45:45.945 32035 32057 D WM-GreedyScheduler: Cancelling work ID fcbd34f2-abe8-40ae-ae07-efc23a911ede
03-30 00:45:45.947 32035 32057 D WM-SystemJobScheduler: Scheduling work ID fcbd34f2-abe8-40ae-ae07-efc23a911edeJob ID 153961
03-30 00:45:46.745 32035 32035 D LocBroadcastReceiver: Received location update.
03-30 00:45:46.748 32035 32035 D ForegrndServiceLauncher: Check if service HighAccuracyLocationService is running. Service running = false
03-30 00:45:46.748 32035 32069 D LocBroadcastReceiver: Last Location: 
03-30 00:45:46.748 32035 32069 D LocBroadcastReceiver: Coords:(49.4829022, 10.9975897)
03-30 00:45:46.748 32035 32069 D LocBroadcastReceiver: Accuracy: 11.625
03-30 00:45:46.748 32035 32069 D LocBroadcastReceiver: Bearing: 0.0
03-30 00:45:46.748 32035 32069 D LocBroadcastReceiver: Begin evaluating if location update should be skipped
03-30 00:45:46.748 32035 32069 D LocBroadcastReceiver: Received location that is 77 milliseconds old, 1743291946671 compared to 1743291946748 with source fused
03-30 00:45:46.752 32035 32069 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:46.761 32035 32069 D LocBroadcastReceiver: Location update sent successfully for 3 as exact
03-30 00:45:50.015 32035 32035 D WebviewActivity: Matter/Thread step changed to NOT_STARTED
03-30 00:45:50.018 32035 32035 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:50.019 32035 32035 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:50.019 32035 32035 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: false, expireMillis: 1743291384220, currentMillis: 1743291950019)
03-30 00:45:50.026 32035 32057 D WM-Processor: Processor cancelling e1391919-34d5-4cdc-8a5c-576427f2da41
03-30 00:45:50.026 32035 32057 D WM-Processor: WorkerWrapper could not be found for e1391919-34d5-4cdc-8a5c-576427f2da41
03-30 00:45:50.027 32035 32057 D WM-GreedyScheduler: Cancelling work ID e1391919-34d5-4cdc-8a5c-576427f2da41
03-30 00:45:50.032 32035 32057 D WM-SystemJobScheduler: Scheduling work ID 9e058c5f-9506-48a2-85b6-7b5cd8d9b7ffJob ID 153962
03-30 00:45:50.036 32035 32057 D WM-GreedyScheduler: Starting tracking for 9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff
03-30 00:45:50.039 32035 32057 D WM-ConstraintTracker: NetworkStateTracker24: initial state = NetworkState(isConnected=true, isValidated=true, isMetered=false, isNotRoaming=true)
03-30 00:45:50.039 32035 32057 D WM-NetworkStateTracker: Registering network callback
03-30 00:45:50.041 32035 32058 D WM-Processor: Processor cancelling fcbd34f2-abe8-40ae-ae07-efc23a911ede
03-30 00:45:50.041 32035 32058 D WM-Processor: WorkerWrapper could not be found for fcbd34f2-abe8-40ae-ae07-efc23a911ede
03-30 00:45:50.041 32035 32148 D WM-NetworkStateTracker: Network capabilities changed: [ Transports: WIFI Capabilities: NOT_METERED&INTERNET&NOT_RESTRICTED&TRUSTED&NOT_VPN&VALIDATED&NOT_ROAMING&FOREGROUND&NOT_CONGESTED&NOT_SUSPENDED&NOT_VCN_MANAGED&NOT_BANDWIDTH_CONSTRAINED LinkUpBandwidth>=15989Kbps LinkDnBandwidth>=35335Kbps TransportInfo: <SSID: <unknown ssid>, BSSID: 02:00:00:00:00:00, MAC: 02:00:00:00:00:00, IP: /192.168.178.51, Security type: 2, Supplicant state: COMPLETED, Wi-Fi standard: 11ax, RSSI: -51, Link speed: 1814Mbps, Tx Link speed: 1814Mbps, Max Supported Tx Link speed: 2401Mbps, Rx Link speed: 1152Mbps, Max Supported Rx Link speed: 2401Mbps, Frequency: 5520MHz, Net ID: -1, Metered hint: false, score: 60, isUsable: true, CarrierMerged: false, SubscriptionId: -1, IsPrimary: -1, Trusted: true, Restricted: false, Ephemeral: false, OEM paid: false, OEM private: false, OSU AP: false, FQDN: <none>, Provider friendly name: <none>, Requesting package name: <none><none>MLO Information: , Is TID-To-Link negotiation supported by the AP: false, AP MLD Address: <none>, AP MLO Link Id: <none>, AP MLO Affiliated links: <none>, Vendor Data: <none>> SignalStrength: -51 UnderlyingNetworks: Null]
03-30 00:45:50.042 32035 32058 D WM-GreedyScheduler: Cancelling work ID fcbd34f2-abe8-40ae-ae07-efc23a911ede
03-30 00:45:50.044 32035 32058 D WM-SystemJobScheduler: Scheduling work ID a852e4a8-12bc-4913-8cd7-840d3aeea0eeJob ID 153963
03-30 00:45:50.045 32035 32058 D WM-GreedyScheduler: Starting work for a852e4a8-12bc-4913-8cd7-840d3aeea0ee
03-30 00:45:50.045 32035 32058 D WM-GreedyScheduler: Starting tracking for 9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff
03-30 00:45:50.046 32035 32056 D WM-Processor: Processor: processing WorkGenerationalId(workSpecId=a852e4a8-12bc-4913-8cd7-840d3aeea0ee, generation=0)
03-30 00:45:50.046 32035 32056 D WM-GreedyScheduler: Constraints met: Scheduling work ID WorkGenerationalId(workSpecId=9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff, generation=0)
03-30 00:45:50.047 32035 32056 D WM-Processor: Processor: processing WorkGenerationalId(workSpecId=9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff, generation=0)
03-30 00:45:50.092 32035 32035 D WM-WorkerWrapper: Starting work for io.homeassistant.companion.android.websocket.WebsocketManager
03-30 00:45:50.092 32035 32035 D WM-WorkerWrapper: Starting work for io.homeassistant.companion.android.sensors.SensorWorker
03-30 00:45:50.093 32035 32035 D WM-SystemJobService: onStartJob for WorkGenerationalId(workSpecId=a852e4a8-12bc-4913-8cd7-840d3aeea0ee, generation=0)
03-30 00:45:50.093 32035 32035 D WM-SystemJobService: onStartJob for WorkGenerationalId(workSpecId=9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff, generation=0)
03-30 00:45:50.094 32035 32057 D WM-Processor: Work WorkGenerationalId(workSpecId=a852e4a8-12bc-4913-8cd7-840d3aeea0ee, generation=0) is already enqueued for processing
03-30 00:45:50.094 32035 32054 D WM-Processor: Work WorkGenerationalId(workSpecId=9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff, generation=0) is already enqueued for processing
03-30 00:45:50.094 32035 32054 I WM-Processor: Moving WorkSpec (9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff) to the foreground
03-30 00:45:50.098 32035 32069 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:50.099 32035 32035 I WM-SystemFgDispatcher: Started foreground service Intent { act=ACTION_START_FOREGROUND cmp=io.homeassistant.companion.android/androidx.work.impl.foreground.SystemForegroundService (has extras) }
03-30 00:45:50.099 32035 32035 D WM-SystemFgDispatcher: Notifying with (id:42, workSpecId: 9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff, notificationType :1)
03-30 00:45:50.099 32035 32069 D SensorWorker: Updating all Sensors in foreground.
03-30 00:45:50.101 32035 32057 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=a852e4a8-12bc-4913-8cd7-840d3aeea0ee, tags={ io.homeassistant.companion.android.websocket.WebsocketManager } ]
03-30 00:45:50.102 32035 32035 D WM-SystemFgDispatcher: Notifying with (id:42, workSpecId: 9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff, notificationType :1)
03-30 00:45:50.103 32035 32069 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:50.105 32035 32035 D WM-Processor: Processor a852e4a8-12bc-4913-8cd7-840d3aeea0ee executed; reschedule = false
03-30 00:45:50.105 32035 32035 D WM-SystemJobService: a852e4a8-12bc-4913-8cd7-840d3aeea0ee executed on JobScheduler
03-30 00:45:50.108 32035 32057 D WM-GreedyScheduler: Cancelling work ID a852e4a8-12bc-4913-8cd7-840d3aeea0ee
03-30 00:45:50.109 32035 32057 D WM-SystemJobScheduler: Scheduling work ID a852e4a8-12bc-4913-8cd7-840d3aeea0eeJob ID 153963
03-30 00:45:50.122 32035 32035 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:50.122 32035 32035 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: false, expireMillis: 1743291384220, currentMillis: 1743291950122)
03-30 00:45:50.130 32035 32035 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:50.130 32035 32035 D ServerConnectionInfo: Using internal URL
03-30 00:45:50.131 32035 32035 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:50.131 32035 32035 D ServerConnectionInfo: Using internal URL
03-30 00:45:50.132 32035 32035 D InsetsController: hide(ime(), fromIme=false)
03-30 00:45:50.132 32035 32035 I ImeTracker: io.homeassistant.companion.android:2d16a8f8: onCancelled at PHASE_CLIENT_ALREADY_HIDDEN
03-30 00:45:50.160 32035 32069 D NextAlarm: Next alarm is scheduled by com.google.android.deskclock with trigger time 1743395400000
03-30 00:45:50.171 32035 32069 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:50.172 32035 32069 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:50.215 32035 32069 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:50.226 32035 32069 I SensorReceiver: Sensor updates and sync completed
03-30 00:45:50.227 32035 32056 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff, tags={ io.homeassistant.companion.android.sensors.SensorWorker } ]
03-30 00:45:50.230 32035 32035 D WM-Processor: Processor 9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff executed; reschedule = false
03-30 00:45:50.230 32035 32035 D WM-GreedyScheduler: Stopping tracking for WorkGenerationalId(workSpecId=9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff, generation=0)
03-30 00:45:50.231 32035 32057 D WM-GreedyScheduler: Cancelling work ID 9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff
03-30 00:45:50.232 32035 32057 D WM-SystemJobScheduler: Scheduling work ID 9e058c5f-9506-48a2-85b6-7b5cd8d9b7ffJob ID 153964
03-30 00:45:50.232 32035 32035 D WM-SystemJobService: 9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff executed on JobScheduler
03-30 00:45:50.232 32035 32035 D WM-SystemFgDispatcher: Removing Notification (id: 42, workSpecId: WorkGenerationalId(workSpecId=9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff, generation=0), notificationType: 1
03-30 00:45:50.233 32035 32035 I WM-SystemFgDispatcher: Stopping foreground service
03-30 00:45:50.233 32035 32035 D WM-SystemFgService: Shutting down.
03-30 00:45:50.238 32035 32035 D WM-SystemJobService: onStopJob for WorkGenerationalId(workSpecId=9e058c5f-9506-48a2-85b6-7b5cd8d9b7ff, generation=0)
03-30 00:45:50.238 32035 32057 D WM-NetworkStateTracker: Unregistering network callback
03-30 00:45:52.573 32035 32147 D WebviewActivity: External bus {"type":"config_screen/show","id":6}
03-30 00:45:52.582 32035 32035 D IntegrationRepository: setAppActive(): false
03-30 00:45:52.583 32035 32035 D IntegrationRepository: setAppActive(): false
03-30 00:45:52.586 32035 32035 D SensorReceiver: Received intent: io.homeassistant.companion.android.UPDATE_SENSORS
03-30 00:45:52.587 32035 32069 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:52.621 32035 32069 D NextAlarm: Next alarm is scheduled by com.google.android.deskclock with trigger time 1743395400000
03-30 00:45:52.629 32035 32069 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:52.630 32035 32069 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:52.646 32035 32035 I ContentCaptureHelper: Setting logging level to OFF
03-30 00:45:52.647 32035 32035 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: false, expireMillis: 1743291384220, currentMillis: 1743291952647)
03-30 00:45:52.671 32035 32069 D SensorReceiver: Nothing to update for server 3 (Home)
03-30 00:45:52.671 32035 32069 I SensorReceiver: Sensor updates and sync completed
03-30 00:45:52.676 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:52.676 32035 32035 W TextView: onProvideContentCaptureStructure(): calling assumeLayout()
03-30 00:45:52.676 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:52.676 32035 32035 W TextView: onProvideContentCaptureStructure(): calling assumeLayout()
03-30 00:45:52.676 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:52.676 32035 32035 W TextView: onProvideContentCaptureStructure(): calling assumeLayout()
03-30 00:45:52.676 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:52.677 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:52.677 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:52.677 32035 32035 W TextView: onProvideContentCaptureStructure(): calling assumeLayout()
03-30 00:45:52.677 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:52.677 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:52.677 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:52.677 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:52.677 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:52.677 32035 32035 W TextView: onProvideContentCaptureStructure(): calling assumeLayout()
03-30 00:45:52.680 32035 32070 D ServerConnectionInfo: usesInternalSsid is: true, usesWifi is: true
03-30 00:45:52.690 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:52.718 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:52.801 32035 32035 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: false, expireMillis: 1743291384220, currentMillis: 1743291952801)
03-30 00:45:52.801 32035 32035 D IntegrationRepository: setAppActive(): true
03-30 00:45:52.803 32035 32035 D InsetsController: hide(ime(), fromIme=false)
03-30 00:45:52.804 32035 32035 I ImeTracker: io.homeassistant.companion.android:82034aaa: onCancelled at PHASE_CLIENT_ALREADY_HIDDEN
03-30 00:45:53.181 32035 32035 D VRI[WebViewActivity]: visibilityChanged oldVisibility=true newVisibility=false
03-30 00:45:53.202 32035 32035 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=false callback=androidx.activity.OnBackPressedDispatcher$Api34Impl$createOnBackAnimationCallback$1@e4a978c
03-30 00:45:54.261 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:54.331 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:54.483 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:55.012 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:55.078 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:55.161 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:55.278 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:55.509 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:55.544 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:55.594 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:55.627 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:55.691 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:55.728 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:55.793 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:55.828 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:55.910 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:55.963 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:56.525 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:56.609 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:56.773 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:57.073 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:57.172 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:57.223 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:57.356 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:57.539 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:57.555 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:57.587 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:57.605 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:57.622 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:57.637 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:57.672 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:57.691 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:59.364 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:59.364 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:59.364 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:45:59.364 32035 32035 E mpanion.android: Invalid resource ID 0x00000000.
03-30 00:46:00.403 32035 32070 D LogcatReader: Read logcat for pid 32035
```

**Screenshot or video of problem:**

**Additional information:**

--- a/common/src/main/kotlin/io/homeassistant/companion/android/common/assist/AssistViewModelBase.kt
+++ b/common/src/main/kotlin/io/homeassistant/companion/android/common/assist/AssistViewModelBase.kt
@@ -98,7 +98,7 @@ abstract class AssistViewModelBase(
                 )
             } else {
                 serverManager.integrationRepository(selectedServerId).getAssistResponse(
-                    text = text!!,
+                    text = text,
                     pipelineId = pipeline?.id,
                     conversationId = conversationId
                 )
@@ -137,7 +137,7 @@ abstract class AssistViewModelBase(
                         val data = (it.data as? AssistPipelineIntentEnd)?.intentOutput ?: return@collect
                         conversationId = data.conversationId
                         continueConversation.set(data.continueConversation)
-                        data.response.speech.plain["speech"]?.let { speech ->
+                        data.response.speech?.plain?.get("speech")?.let { speech ->
                             onEvent(AssistEvent.Message.Output(speech))
                         }
                     }

--- a/common/src/main/kotlin/io/homeassistant/companion/android/common/data/websocket/impl/entities/ConversationSpeechPlainResponse.kt
+++ b/common/src/main/kotlin/io/homeassistant/companion/android/common/data/websocket/impl/entities/ConversationSpeechPlainResponse.kt
@@ -4,5 +4,5 @@ import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 data class ConversationSpeechPlainResponse(
-    val plain: Map<String, String?>
+    val plain: Map<String, String?>?
 )

--- a/common/src/main/kotlin/io/homeassistant/companion/android/common/data/websocket/impl/entities/ConversationSpeechResponse.kt
+++ b/common/src/main/kotlin/io/homeassistant/companion/android/common/data/websocket/impl/entities/ConversationSpeechResponse.kt
@@ -4,7 +4,7 @@ import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 data class ConversationSpeechResponse(
-    val speech: ConversationSpeechPlainResponse,
+    val speech: ConversationSpeechPlainResponse?,
     val card: Any?,
     val language: String?,
     val responseType: String?,

